org.osgi.security.sysatt110.test/src/org/osgi/security/sysatt110/test/SysAtt110Test.java

```diff
@@ -1,54 +1,47 @@
 package org.osgi.security.sysatt110.test;
 
-import org.osgi.framework.BundleContext;
+import static org.osgi.framework.FrameworkUtil.getBundle;
+
+import java.io.File;
+
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
-import org.osgi.framework.BundleActivator;
-
+import org.osgi.framework.BundleContext;
 import org.osgi.security.util.api.Util;
 import org.osgi.util.tracker.ServiceTracker;
 
-import java.io.File;
-
 /**
  * 
  **/
 
-public class Activator implements BundleActivator
+public class SysAtt110Test
 {
-	private static BundleContext bundleContext;
-	private ServiceTracker serviceRef;
+	private BundleContext bundleContext = getBundle(SysAtt110Test.class).getBundleContext();
```

Comment:
I think you just need to add a public setBundleContext(BundleContext) method which the bnd osgi junit launcher will call to inject the bundle context. See https://github.com/bndtools/bnd/blob/master/biz.aQute.junit/src/aQute/junit/BasicTestReport.java#L56-L84